pkg/grizzly/providers.go

```diff
@@ -231,7 +231,9 @@ func (r *Registry) HandlerMatchesTarget(handler Handler, targets []string) bool
 			return true
 		}
 	}
-	r.Notifier().Info(SimpleString(key), "skipped")
+	if !isSilent {
```

Comment:
Where is this false? And shouldn't this log message move there then instead?

Reply:
yeah, that's actually fair. Notifying shouldn't be a side-effect of an "Is" function.